src/api/convert-html.js

```diff
@@ -0,0 +1,77 @@
+const { Collection, Item, Request, Response } = require('postman-collection');
+const PostmanHTMLExtraReporter = require('newman-reporter-htmlextra');
+const { logger } = require('../utils/logger');
+
+function generateHTMLReport(summary) {
+    try{
+        logger.info(
+            `Starting the conversion of JSON summary to HTML report for collection '${summary.collection.info.name}'.`
+        );
+
+        var collection = new Collection(summary.collection.toString());
+        collection.name = summary.collection.info.name;
+      
```

Comment:
Add a comment to explain we need to refill some of the function/item lost during serialization